client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -1,6 +1,10 @@
-import { uniqWith } from 'lodash';
 import { useState, useEffect } from 'react';
+import { useMutation } from 'react-query';
```

Comment:
the changes here are simply reordering - convention has it that 'react' imports are first, and I've also moved 'react-query' up a bit. 
The preferred order (for me, at least) is:
- react
- external packages
- internal packages
- local files

so that's what I've done. no functional changes.